sample-ms/src/main/java/net/trajano/ms/example/TestCallback.java

```diff
@@ -0,0 +1,22 @@
+package net.trajano.ms.example;
+
+import javax.ws.rs.container.CompletionCallback;
+import javax.ws.rs.ext.Provider;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Provider
+public class TestCallback implements
+    CompletionCallback {
+
+    private static final Logger LOG = LoggerFactory.getLogger(TestCallback.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)